packages/design/src/DataTable/types.ts

```diff
@@ -1,9 +1,13 @@
+import { MatchCallback } from 'design/utils/match';
+
 export type TableProps<T> = {
   data: T[];
   columns: TableColumn<T>[];
   emptyText: string;
   pagination?: PaginationConfig;
   isSearchable?: boolean;
+  searchableProps?: Extract<keyof T, string>[];
+  customSearchMatchers?: MatchCallback<T>[];
```

Comment:
lets add back the comment explaining that searchValue comes transformed to lower case